pkgs/tools/backup/rdup/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchzip, pkgconfig, autoreconfHook, glib, pcre }:
+
+stdenv.mkDerivation rec {
+  name = "rdup-${version}";
+  version = "1.1.15";
+
+  src = fetchzip {
```

Comment:
Please use `fetchFromGitHub` or `fetchgit` (if submodules need to be cloned) otherwise there are periodic hash changes due to regeneration of the github zip files.